lib/roby/app/rake.rb

```diff
@@ -319,6 +324,10 @@ def run_roby_test(*args, report_name: "report")
                         FileUtils.mkdir_p report_dir
                     end
 
+                    args += test_files.flat_map do |file|
```

Comment:
test_files is not initialized, i.e. is is nil by default.

Also, `array.flat_map { [value] }` is equivalent to `array.map { value }`

Reply:
fixed

Reply:
Nothing changed ... pushed the wrong commit ?

Reply:
I pushed in the wrong branch, probably I need a coffee